controllers/src/main/java/com/redhat/ipaas/controllers/integration/IntegrationController.java

```diff
@@ -24,39 +31,29 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.stereotype.Service;
 
-import javax.annotation.PostConstruct;
-import javax.annotation.PreDestroy;
-import java.io.IOException;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Optional;
-import java.util.concurrent.*;
-
 /**
  * This class tracks changes to Integrations and attempts to process them so that
  * their current status matches their desired status.
  */
 @Service
 public class IntegrationController {
-    private static final Logger LOGGER = LoggerFactory.getLogger(IntegrationController.class.getName());
+    private final Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
Don't worry about it. I prefer the instance field approach too, it allows for changing log level per instance, rather than at the logger name as per normal.